packages/v2/package.json

```diff
@@ -7,7 +7,7 @@
     "xbuild": "vite build --emptyOutDir",
     "build": "vite build --emptyOutDir && npm run deploy",
     "serve": "vite preview",
-    "deploy": "bash -c '../../scripts/make_header.sh ../../_static/v2 server_index.h web_server'"
+    "deploy": "bash -c '../../scripts/make_header.sh ../../_static/v2 server_index.h web_server INDEX_GZ'"
```

Comment:
I adapted this change from the original pull request, but I am not familiar with what it is doing or why it is required.